Error occurred while running FullExample

Running with version 4.12.0, full error stack:

```shell script
io.fabric8.kubernetes.client.KubernetesClientException: An error has occurred.
	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:64)
	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:53)
	at io.fabric8.kubernetes.client.dsl.internal.apps.v1.RollableScalableResourceOperation.edit(RollableScalableResourceOperation.java:163)
	at io.fabric8.kubernetes.client.dsl.internal.apps.v1.RollableScalableResourceOperation.edit(RollableScalableResourceOperation.java:44)
	at io.fabric8.kubernetes.examples.FullExample.main(FullExample.java:156)
Caused by: java.lang.NoSuchMethodException: io.fabric8.kubernetes.api.model.DoneableReplicationController.<init>(io.fabric8.kubernetes.api.model.ReplicationController, io.fabric8.kubernetes.api.builder.Visitor)
	at java.lang.Class.getConstructor0(Class.java:3082)
	at java.lang.Class.getDeclaredConstructor(Class.java:2178)
	at io.fabric8.kubernetes.client.dsl.internal.apps.v1.RollableScalableResourceOperation.edit(RollableScalableResourceOperation.java:161)
	... 2 more
[main] ERROR io.fabric8.kubernetes.examples.FullExample - An error has occurred.
io.fabric8.kubernetes.client.KubernetesClientException: An error has occurred.
	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:64)
	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:53)
	at io.fabric8.kubernetes.client.dsl.internal.apps.v1.RollableScalableResourceOperation.edit(RollableScalableResourceOperation.java:163)
	at io.fabric8.kubernetes.client.dsl.internal.apps.v1.RollableScalableResourceOperation.edit(RollableScalableResourceOperation.java:44)
	at io.fabric8.kubernetes.examples.FullExample.main(FullExample.java:156)
Caused by: java.lang.NoSuchMethodException: io.fabric8.kubernetes.api.model.DoneableReplicationController.<init>(io.fabric8.kubernetes.api.model.ReplicationController, io.fabric8.kubernetes.api.builder.Visitor)
	at java.lang.Class.getConstructor0(Class.java:3082)
	at java.lang.Class.getDeclaredConstructor(Class.java:2178)
	at io.fabric8.kubernetes.client.dsl.internal.apps.v1.RollableScalableResourceOperation.edit(RollableScalableResourceOperation.java:161)
	... 2 more
```

--- a/kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/FullExample.java
+++ b/kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/FullExample.java
@@ -153,7 +153,7 @@ public class FullExample {
 
                 // Update the RC via rolling update with inline builder
                 client.replicationControllers().inNamespace("thisisatest").withName("nginx-controller")
-                        .rolling().edit().editMetadata().addToLabels("testing", "rolling-update").endMetadata().done();
+                        .rolling().updateImage("nginx:stable-alpine");
 
                 Thread.sleep(1000);
 

